### PR TITLE
SOC-5289: Accessing a page having space name display it in space navigations

### PR DESCRIPTION
**Pull request by boueslati**
Ensure that the URL is from group site otherwise, it will return NULL space.

### CODE REVIEW

File: component/webui/src/main/java/org/exoplatform/social/webui/Utils.java [hunk 1]
```diff
@@ -813,6 +813,11 @@ public static Space getSpaceByContext() {
       String groupId = String.format("%s/%s", SpaceUtils.SPACE_GROUP, spacePrettyName);
       space = spaceService.getSpaceByGroupId(groupId); 
     }
+
+    if(!pcontext.getControllerContext().getParameter(RequestNavigationData.REQUEST_SITE_TYPE).equals("group")) {
```
Comment (thomasdelhomenie) [inline component/webui/src/main/java/org/exoplatform/social/webui/Utils.java]: Shouldn't this test be the first of the method (not a group navigation -> cannot be a space) ? Or is there cases where we can be in a space without being in a group navigation ?

Comment (thomasdelhomenie) [inline component/webui/src/main/java/org/exoplatform/social/webui/Utils.java]: Don't we have a constant for "group" navigation type ?

Comment (boueslati) [inline component/webui/src/main/java/org/exoplatform/social/webui/Utils.java]: > Or is there cases where we can be in a space without being in a group navigation ?

That's what I supposed. So, I let the behavior as it was, and just before returning the space, I check if it's a group navigation. Of course, it's more meaningful to check it before calling space service and getting the space.
> Don't we have a constant for "group" navigation type ?

I think it's SiteType.GROUP.getName() , isn't it?

Comment (thomasdelhomenie) [inline component/webui/src/main/java/org/exoplatform/social/webui/Utils.java]: > That's what I supposed. So, I let the behavior as it was, and just before returning the space, I check if it's a group navigation. Of course, it's more meaningful to check it before calling space service and getting the space.

Actually, with this code, if it is not a group navigation, it returns null even if we retrieved a space in the previous block (with getSpaceByPrettyName or getSpaceByGroupId). So I think this new test should at least be moved above (so after the block of "route == null").

> I think it's SiteType.GROUP.getName() , isn't it?

Looks good

**Pull request closed without merging**